lib/std/io.zig

```diff
@@ -182,10 +191,10 @@ pub const BufferedAtomicFile = @import("io/buffered_atomic_file.zig").BufferedAt
 pub const StreamSource = @import("io/stream_source.zig").StreamSource;
 
 /// A Writer that doesn't write to anything.
-pub const null_writer = @as(NullWriter, .{ .context = {} });
+pub var null_writer = @as(NullWriter, .{ .context = {} });
```

Comment:
Can you talk a little bit about the idea here to change streams from being passed by-val? Why is this necessary?

Reply:
Oh, that's a left over as it turns out, I'll revert it. As the auto_indenting_stream needs to be non-const there were some awkward const dependencies going on.